src/mozillavpn.cpp

```diff
@@ -92,6 +94,7 @@
 #include <QApplication>
 #include <QBuffer>
 #include <QDir>
+#include <QEvent>
```

Comment:
can this also go up with `#include <QFileOpenEvent>` under the "only for macos" stuff?